docs/book/v2/table.md

```diff
@@ -9,25 +9,26 @@ For reasons that will become clear presently, we recommend creating memory
 tables by extending the `Swoole\Table` class, defining the appropriate columns
 and table size inside of the constructor.
 
-> ### Initialize the table within the constructor
+> ### Initialize the Table within the Constructor
```

Comment:
```suggestion
> ### Initialize the Table Within the Constructor
```
If you're going for title casing, `Within` should also have an initial capital.

Reply:
I have interpreted "within" as preposition here. I will fix this.